src/55functions.js

```diff
@@ -157,7 +157,17 @@ stdlib.LOWER = stdlib.LCASE = function(s) {return und(s,'String(y).toLowerCase()
 //stdlib.LCASE = function(s) {return '('+s+').toLowerCase()';}
 
 
-// LTRIM
+// Returns a character expression after it removes leading blanks.
+// see https://docs.microsoft.com/en-us/sql/t-sql/functions/ltrim-transact-sql
+stdlib.LTRIM = function(s) {
+    return und(s, 'y.substr(y.indexOf(y.trim()), y.length)');

```

Comment:
I actually did that first and was really confused why it didn't work. (thinking making regex couldn't be used in the alasql engine).

But after a long time... I figure out the `\s` needed to be escaped to become `\\s`. I guess for these functions there passed through `eval` statement at one point?